README.md

```diff
@@ -4,10 +4,11 @@
 <img src="https://github.com/containerd/containerd/blob/master/docs/images/containerd-dark.png" width="200" >
 </p>
 
-Note: `cri-containerd` is transitioning from a standalone binary that talks to
-containerd to a plugin within containerd. This github branch is for the `cri`
-plugin. See [cri-containerd standalone branch](https://github.com/containerd/cri/tree/standalone-cri-containerd)
-for information about the standalone version of cri-containerd.
+*Note: The standalone `cri-containerd` binary is deprecated. `cri-containerd` is
```

Comment:
deprecated indicates that we shipped a supported version..

Reply:
We did ship supported versions in 1.0.0-alpha.x and 1.0.0-beta.x.

Reply:
Support usually carries with it a ga announcement, a branch for the GA, a promise that there will be no breaking changes, a declaration that the product is production ready etc..  We don't deprecate "products" so much as we deprecate behaviors of products.

Reply:
Changed to end-of-life.